src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -133,20 +142,19 @@ class BlockingProcess(
     val blockingIODispatcherId = context.system.settings.config.getString(BlockingIODispatcherId)
 
     try {
-      val selfRef = context.self
       val selfDispatcherAttribute = ActorAttributes.dispatcher(blockingIODispatcherId)
 
       val stdin = StreamConverters.fromOutputStream(() => process.getOutputStream())
         .withAttributes(selfDispatcherAttribute)
-        .mapMaterializedValue(_.andThen { case _ => selfRef ! StdinTerminated })
+        .mapMaterializedValue(_.andThen { case _ => self ! StdinTerminated })
```

Comment:
I think this is okay - see https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/actor/Actor.scala#L484 which shows `self` is a final immutable val